test/serializer/abstract/Delete1.js

```diff
@@ -1,5 +1,4 @@
-// throws introspection error
-var obj = global.__abstract ? __abstract("object", "({ p: 41} )") : { p: 41 };
+var obj = global.__makePartial ? __makePartial({ p: 41 }) : { p: 41 };
```

Comment:
I think the point of this test case was to make sure we fail nicely in case of an attempt to delete an *unknown* property on a partial object.

Reply:
That is the job of Delete.js.